front/pages/api/stripe/webhook.ts

```diff
@@ -171,6 +171,36 @@ async function handler(
           // The stripe subscription becomes "past_due".
           // We keep active and email the user and us to manually manage those cases first?
           break;
+        case "customer.subscription.updated":
```

Comment:
If we don't do anything, why is it not treated like other events we don't handle ? Ie do nothing as part of `default:` ?

Reply:
I added some loggers. To start with, I'd like to be notified when the event occurs to make sure there is not a big hole in the racket